typings/index.d.ts

```diff
@@ -41,16 +40,15 @@ export default class Client {
   updateFile(fileID: ID, params: FileChange): Promise<File>
   deleteFile(fileID: ID): Promise<{}>
   listLinkedFiles(): Promise<Array<LinkedFile>>
-  createLabel(name: string, color: string): Promise<any>
-  listLabels(): Promise<Array<Label>>
   createLinkedFile(params: LinkedFileChange): Promise<LinkedFile>
   deleteLinkedFile(linkedFileID: ID): Promise<{}>
+  createLabel(name: string, color: string): Promise<any>
+  listLabels(): Promise<Array<Label>>
   listTeams(): Promise<Array<Team>>
   getTeam(teamID: ID): Promise<Team>
   listIterations(): Promise<Array<Iteration>>
   createIteration(params: IterationChange): Promise<Iteration>
   getIteration(iterationID: ID): Promise<Iteration>
-  updateIteration(iterationID: ID, params: IterationChange): Promise<Iteration>
```

Comment:
Hmm, the script to generate TypeScript definitions doesn't like breaking lines.

This is not related to this PR, I'll send a PR to fix this.

Reply:
OK! I wont release this. Mind doing the release after your PR?